extensions/commands/src/org/exist/commands/threads/Deadlock.java

```diff
@@ -45,7 +47,8 @@ public Deadlock() {
 	@Override
 	public void process(XmldbURI collection, String[] commandData) throws CommandException {
 		
-        Map<String, LockInfo> threads = DeadlockDetection.getWaitingThreads();
+		//TODO:check where that method is
+        Map<String, LockInfo> threads = Collections.emptyMap(); // DeadlockDetection.getWaitingThreads();
```

Comment:
As per previous comment...

Reply:
Follow up problem of not existing import